ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -124,6 +124,8 @@
 {% include 'eos/loopback-interfaces.j2' %}
 {# Management Interfaces #}
 {% include 'eos/management-interfaces.j2' %}
+{# Tunnel Interfaces #}
```

Comment:
please ensure this matches the order of EOS config. Have not tested, so just resolve if ok.

Reply:
I meant where in the config they are. So are you importing after management interfaces because they are created there in EOS?

Reply:
Yeah sorry maybe my answer was not explicit enough..

EOS sorts the interfaces by alphabetical order and so I checked while writing the PR to see where it should be inserted:

```
interface Management1
   description toto
   vrf mgmt
   ip address 10.81.121.163/24
   no lldp transmit
   no lldp receive
interface Tunnel10
   ipv6 enable
   ipv6 address dead:beef::1/64
interface Tunnel42
interface Vlan42
   multicast ipv4 source route export
interface Vlan444
   ipv6 address virtual 2001::1/64
interface Vlan555
   ip address virtual 43.42.42.42/24
```